deploy/cr.yaml

```
@@ -3,6 +3,7 @@ kind: PerconaServerMongoDB
 metadata:
   name: my-cluster-name
 #  finalizers:
+#    - delete-psmdb-in-order
```

1. please enable it by default
2. please rename finalizer
```suggestion
  finalizers:
    - delete-psmdb-pods-in-order
```

---

fixed in d63633e678f9b40252449f0704914f4cde22b716